scripts/tests/genesis.json

```
@@ -12,10 +12,15 @@
     "berlinBlock": 0,
     "londonBlock": 0,
     "mergeForkBlock": 0,
+    "shanghaiTime": 0,
+    "shardingForkTime": 0,
```

Looks like this key should be called `cancunTime`.

```suggestion
    "cancunTime": 0,
```